segments/battery.sh

```diff
@@ -39,6 +41,9 @@ run_segment() {
 	"cute")
 		output=$(__cutinate "$battery_status")
 		;;
+	"hearts")
+		output=$(__generate_hearts "$battery_status")
```

Comment:
```suggestion
		output=$(__generate_hearts "${battery_status/* /}")
```

While testing, i found out, that `$battery_status` is already containing a battery symbol. This is unfortunately baked into all the sub functions directly and needs a bigger rework. For now see my suggestion above.